analyzer/codechecker_analyzer/analyzers/clangsa/ctu_manager.py

```diff
@@ -155,9 +155,16 @@ def func_map_list_src_to_ast(func_src_list, ctu_on_demand):
 
     func_ast_list = []
     for fn_src_txt in func_src_list:
-        dpos = fn_src_txt.find(" ")
-        mangled_name = fn_src_txt[0:dpos]
-        path = fn_src_txt[dpos + 1:]
+        if fn_src_txt[0].isdigit():
+            length_str, _ = fn_src_txt.split(':', 1)
```

Comment:
I'm sure it's quite unlikely, but can we make sure that the line contains exactly one colon? I mean can the path be in Windows format? (e.g. C:\blabla)

Reply:
The format is `<Length>:<USR> <File-Path>`. My understanding is that `split(':', 1)` would create a tuple with two elements. The first element will be the string until the **first** colon. In that sense it doesn't matter if we have a colon in the path, because that is placed in the second element of the tuple.